docs/google/devices.md

```diff
@@ -84,6 +84,45 @@ using a error rate of 0.5% per gate, a circuit of depth 20 and width 20 could be
 at 0.995^(20*20) = 0.135. Using separate error rates per gates (i.e. based on calibration
 metrics) or a more complicated noise model can result in more accurate error estimation.
 
+### Use sweeps when possible
+
+Round trip network time to and from the engine typically adds latency on the order of a second
+to the overall computation time.  Reducing the number of trips and allowing the engine to
+properly batch circuits can improve the throughput of your calculations.  One way to do this
+is to use parameter sweeps to send multiple variations of a circuit at once.
+
+One example is to turn single-qubit gates on or off by using parameter sweeps.  
+For instance, the following code creates a single circuit that parameterizes
+the decision of measuring the |0> or the |1> state.  This code uses a simulator
```

Comment:
A more relevant example would be Z axis vs X axis basis measurement. Flipping the bit is easy to do classically in the post-processing.

Here's my take:

```python
import cirq
import sympy
q = cirq.GridQubit(1, 1)
sampler = cirq.Simulator()

# STRATEGY #1: Have a separate circuit and sample call for each basis.
circuit_z = cirq.Circuit(
    cirq.measure(q, key='out'))
circuit_x = cirq.Circuit(
    cirq.H(q),
    cirq.measure(q, key='out'))
samples_z = sampler.sample(circuit_z, repetitions=5)
samples_x = sampler.sample(circuit_x, repetitions=5)

print(samples_z)
# prints something like:
#    out
# 0    0
# 1    0
# 2    0
# 3    0
# 4    0

print(samples_x)
# prints something like:
#    out
# 0    0
# 1    1
# 2    1
# 3    0
# 4    0

# STRATEGY #2: Have a parameterized circuit.
circuit_sweep = cirq.Circuit(
    cirq.H(q)**sympy.Symbol('t'),
    cirq.measure(q, key='out'))

samples_sweep = sampler.sample(circuit_sweep,
                               repetitions=5,
                               params=[{'t': 0}, {'t': 1}])
print(samples_sweep)
# prints something like:
#    t  out
# 0  0    0
# 1  0    0
# 2  0    0
# 3  0    0
# 4  0    0
# 0  1    0
# 1  1    1
# 2  1    1
# 3  1    0
# 4  1    1
```

Reply:
LGTM, I like this better.  I took your code as verbatim.